spread.yaml

```diff
@@ -14,6 +15,8 @@ include:
     - requirements-focal.txt
     - Makefile
     - rockcraft/
+    - sphinx-starter-pack/
+    - .git/
```

Comment:
include `.git`? how come?

Reply:
which spread test calls `make prepare-docs`?

Reply:
the how-to/code/build-docs/task.yaml with ``make installdocs``

Reply:
Do you have an idea about it without importing the .git folder ?